pages/common/mosquitto_sub.md

```diff
@@ -0,0 +1,15 @@
+# mosquitto_sub
+
+> A simple MQTT version 3.1.1 client that will subscribe to topics and print the messages that it receives.
+
+- Subscribe to all broker status messages publishing on `iot.eclipse.org` port 1885 and print published messages verbosely, by default hostname resolves to `localhost` and port `1883`:
+
+`mosquitto_sub -v -h "iot.eclipse.org" -p 1885 -t {{\$SYS/#}}`
+
+- Subscribe to the topic `sensors/temperature` information on `localhost` with Quality of Service (`QoS`) set to 1:
```

Comment:
I think this can be the first example, since this is short. And also mention the default port of 1883 here, so that you don't have to repeat that info in the above example.